Nest/00. Plan.md

```diff
@@ -0,0 +1,67 @@
+# 1 Introduction
+## 1.1 Package manager
+
+## 1.2 Packages and dependencies
+
+## 1.3 Repositories
+
+## 1.4 Package's syntax
+
+# 2 Nest
+## 2.1 Nest's options
+
+## 2.2 Nest commands
+### 2.2.1 Synchronizing the local cache
+### 2.2.2 Troubleshooting
+
+## 2.3 Installing and uninstalling a package
+### 2.3.1 Installing a package
+### 2.3.2 Uninstalling a package
+### 2.3.3 Troubleshooting
+
+## 2.4 Updating your system
+### 2.4.1 Troubleshooting
+
+## 2.5 Conclusion
+
+# 3 Some more advanced concepts
+## 3.1 More about the repositories
+### 3.1.1 The official repositories
+### 3.1.2 The unofficial repositories
+### 3.1.3 Changing the repositories' configuration file
+
+## 3.2 The cache
+
+## 3.3 Version requirement
+
+# 4 Finest
+## 4.1 Key concepts
+### 4.1.1 The cache
+### 4.1.2 The transactions
+### 4.1.3 The dependency graph
+### 4.1.4 Groups
+### 4.1.5 Example
```

Comment:
Example of what? (I'd be more precise)

Reply:
`Example of transactions with the dependency graph`?

Reply:
What about "Dependency graph examples" ? I think the transactions themselves aren't the focus themselves from what I'm gathering ?

Reply:
`Dependency graph examples` makes me think that there will be literally examples of static dependency graphs and that's it
I think the idea of transactions in the title is still important, like maybe `Example of how transactions are altering the dependency graph`, even though it's a tiny bit long, it's pretty accurate

Reply:
"Dependency graph transaction examples" could potentially work to.
Also for your suggestion you can get rid of most of the verbs:
"Example of transactions altering the dependency graph"

Reply:
I prefer the first title you proposed. But I would rephrase it as `Example: dependency graph transactions`, in order to really emphasize on the word `Example`